src/accounts/__tests__/deleteAccount.test.js

```diff
@@ -0,0 +1,61 @@
+const { dbConnect, dbCleanUP, dbDisconnect } = require('../../../test/test.db');
+const app = require('../../../app');
+const supertest = require('supertest');
+const api = supertest(app);
+const helper = require('../../../test/testHelper');
+let token;
+jest.setTimeout(100000);
+beforeAll(async () => {
+  await dbConnect();
+
+  const { accounts } = helper;
+
+  // create account in database
+  await Promise.all(accounts);
+
+  const user = helper.accountsAsJson[0];
+  await login(user);
+});
+
+afterAll(async () => {
+  await dbCleanUP();
+  await dbDisconnect();
+});
+
+describe('Delete Account', () => {
+  it('should return error when password is missing', async () => {
+    const response = await api
+      .delete('/accounts/delete-account')
+      .set('Authorization', `Bearer ${token}`);
+
+    expect(response.status).toBe(422);
+    // expect(response.body.message).toBe("\"password\" is required");
+  });
+
+  it('should "delete" an account and return isDeleted as true', async () => {
+    const response = await api
+      .delete('/accounts/delete-account')
+      .set('Authorization', `Bearer ${token}`)
+      .send({ password: '8XLxWgzXSPh9ABS!' });
```

Comment:
Let us try to avoid hard-coded, literal values

Reply:
I'll correct that, but for the case where I am trying to pass a wrong password can I use that or create a variable for it and reference